www/favicons/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/favicons/mstile-150x150.png"/>
+            <TileColor>#603cba</TileColor>
```

Comment:
That seems like an odd choice of `TileColor` … lavender?

![image](https://user-images.githubusercontent.com/96150/113913066-b99b9c80-9790-11eb-817a-dc27913e32b9.png)

Reply:
It was one of a couple of choices in the wizard, possibly based on some set of Windows defaults. Should we go with white for now?